CONTRIBUTING.md

```diff
@@ -13,15 +13,17 @@ branch only.
 ### Building Development Versions
 
 To build development versions, first you need a configuration file in
-`~\sublime-package-dev.json`.
+`~\.sublime-package-dev.json`.
 
 This file must contain the following information:
 
     {
-        "pathToSublimeText": "...\sublime_text.exe",
-        "pathToSublimeTextData": "...\
+        "pathToSublimeText": "...\\sublime_text.exe",
```

Comment:
I was trying to use this on Windows and it didn't work until I switched it to double back-slash.